Cloudflare/network_gateway/ingest/parser.yml

```
@@ -0,0 +1,32 @@
+name: cloudflare-network-gateway
+pipeline:
+  - name: json_event
+    external:
+      name: json.parse-json
+      properties:
+        input_field: "{{original.message}}"
+  - name: set_ecs_fields
+  - name: set_cloudflare_fields
+
+stages:
+  set_ecs_fields:
+    actions:
+      - set:
+          "@timestamp": '{{json_event.message.Datetime}}'
+          destination.ip: '{{json_event.message.DestinationIP}}'
+          destination.port: '{{json_event.message.DestinationPort}}'
+          device.id: '{{json_event.message.DeviceID}}'
+          event.category: ["network"]
+          event.dataset: "network_gateway"
+          event.kind: 'event'
```

I suggest adding the following fields:
```suggestion
          event.kind: 'event'
          observer.type: "proxy"
          observer.vendor: "Cloudflare"
```